lib/ansible/config/ansible_builtin_runtime.yml

```diff
@@ -2160,24 +2160,24 @@ plugin_routing:
       redirect: community.network.exos_lldp_interfaces
     exos_vlans:
       redirect: community.network.exos_vlans
-    bigip_asm_policy:
```

Comment:
What's the state of this, does it need tombstoning?

Reply:
yeah it is tombstoned, fixing this now